CHANGELOG.md

```diff
@@ -16,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 * Output directory logic for the `run` command now always uses the workflow or task name by default ([#148](https://github.com/stjude-rust-labs/sprocket/pull/148)).
+    * this logic was replaced in [#149](https://github.com/stjude-rust-labs/sprocket/pull/149).
```

Comment:
Should delete the previous entry; we aren't obligated to keep anything in this changelog until it's part of a release.